### ISSUE TITLE
Does not work well with Pest, a testing framework

### ISSUE THREAD

**Issue by schrosis** [DEVSENSE/phptools-docs#368]
For example, in the sample code created when [install Pest](https://pestphp.com/docs/installation).

```php
<?php

test('that true is true', function () {
    expect(true)->toBeTrue();
});

```

## 1. Generics of the `expect()` function

Cannot infer generics, and the following error occurs.

> Argument '1' passed to expect() is expected to be of type TValue|null, bool given PHP(PHP0406)

## 2. Method autocompletion

When I try to autocompletion expectation methods such as `toBe()`, it takes a very long time. Or the language server shutdown.

- - - 

VSCode version: 1.80.2
Extension version: v1.36.13417
PHP version: 8.2.8

**Comment by schrosis** [comment]
I checked and it autocompletes very comfortably!
Thank you for fixing No2.

**Comment by jakubmisek** [comment]
Thank you for testing so quickly, I'm glad `2` works now.

We'll keep working on `1`.

**Comment by jakubmisek** [comment]
`1` has been fixed; available in the next update (the one after 1.38.13874).

thank you!

**Comment by jakubmisek** [comment]
fixed in the latest pre-release! thank you
